modules/xfem/doc/content/source/dirackernels/XFEMFunctionNeumannBC.md

```diff
@@ -0,0 +1,17 @@
+# XFEMFunctionNeumannBC
+
+!syntax description /DiracKernels/XFEMFunctionNeumannBC
+
+## Overview
+
+`XFEMFunctionNeumannBC` applies a Neumann BC at an interface cut by XFEM. The value of the BC is provided by a `Function`.
+
```

Comment:
`XFEMFunctionNeumannBC` applies a Neumann BC at both sides of XFEM cut line/surface.

Reply:
Yes -- that's what I need for now. Maybe in the future one can extend `XFEMIntegratedBC` to optionally apply the BC to only one side of the interface?

Reply:
Oh you are suggesting a change. I misunderstood...